src/MassTransit/Configuration/Configuration/IHostConfiguration.cs

```diff
@@ -15,6 +16,8 @@ public interface IHostConfiguration :
         ISendObserverConnector,
         ISpecification
     {
+        IAgent Agent { get; }
```

Comment:
This won't last, at all.

Reply:
any suggestion? I need it to stop rider when Bus stopped